src/main/java/unicash/commons/enums/CommandType.java

```diff
@@ -232,7 +232,7 @@ public String getMessageUsage() {
 
         @Override
         public String getMessageSuccess() {
-            return "Found %1$d transactions:\n\n%2$s";
+            return "%1$d transactions listed!";
```

Comment:
Is this not a message change? :o

Reply:
The "new" message is the current message displayed and taken directly from the `FindCommand.java` file, changed it so that there would be no change in behaviour actually

Reply:
@jamz903 Nicely done, I was thinking of adding the getMessageSuccess to find but I thought it would be a message change, didn't think to change CommandType itself